packages/snet_cli/snet/snet_cli/mpe_service_metadata.py

```diff
@@ -40,66 +42,110 @@
 import secrets
 
 from collections import defaultdict
+from enum import Enum
 
 from snet.snet_cli.utils import is_valid_endpoint
 
 
+# Supported Asset types
+class AssetType(Enum):
+    HERO_IMAGE = "hero_image"
```

Comment:
Lets include a type for documentation and Terms of Use

Reply:
added new asset types